tests/travis/_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Is there a reason you didn't use https://github.com/silverstripe-labs/silverstripe-travis-support? Its standard across our other maintained modules, and allows for things like switching the core dependency easily.